app/flows/fluggastrechteVorabcheck/stringReplacements.ts

```diff
@@ -114,16 +117,36 @@ export function getRouteCompensationDescription(
   };
 }
 
-function getAirportName(airportIataCode: string): string {
-  if (airportIataCode.length > 0) {
-    const airport = airports.find((aiport) => aiport.iata === airportIataCode);
-
-    if (airport) {
-      return airport.airport.includes(airport.city)
-        ? `${airport.airport} (${airport.iata})`
-        : `${airport.city} ${airport.airport} (${airport.iata})`;
-    }
+function getAirlineByIataCode(iataCode?: string) {
+  if (typeof iataCode === "undefined" || iataCode.length === 0) {
+    return undefined;
   }
 
-  return "";
+  return airlines.find((airline) => airline.iata === iataCode);
+}
+
+export function hasSchlichtungsstelleBfJ({
+  fluggesellschaft,
+}: FluggastrechtVorabcheckContext) {
+  const airline = getAirlineByIataCode(fluggesellschaft);
+
+  return {
+    hasSchlichtungsstelleBfJ:
+      typeof airline === "undefined" ||
+      airline?.arbitrationBoard === SCHLICHTUNGSSTELLE_LUFTVERKEHR_CODE ||
```

Comment:
can we align and use English for this? if someone new to this project, they would need longer time to understand this
`ARBITRATION_BOARD_BFJ` and `ARBITRATION_BOARD_SOEP`

```suggestion
    hasArbitrationBoardBfJ:
      typeof airline === "undefined" ||
      airline?.arbitrationBoard === ARBITRATION_BOARD_BFJ ||
```